packages/plugin-ext-vscode/src/browser/plugin-vscode-commands-contribution.ts

```diff
@@ -101,6 +108,20 @@ export class PluginVscodeCommandsContribution implements CommandContribution {
             }
         });
 
+        commands.registerCommand(VscodeCommands.OPEN_FOLDER, {
+            isVisible: () => false,
+            execute: async (resource?: URI, options?: WorkspaceInput | undefined) => {
```

Comment:
I'm not suer about `WorkspaceInput`.

We should use VS Code type definition for options and translate it to `WorkspaceInput`. Could you look up the relevant code in VS Code?

Reply:
vscode openFolder api - https://github.com/microsoft/vscode/blob/2af91664d482c6065f7a0d589823ec115a65ec10/src/vs/workbench/api/common/apiCommands.ts#L43